platform.md

```diff
@@ -671,6 +686,7 @@ User-provided modifications to the process execution environment SHOULD be set d
 
 The process SHALL inherit both stack-provided and user-provided variables from the lifecycle execution environment with the following exceptions:
 * `CNB_APP_DIR`, `CNB_LAYERS_DIR` and `CNB_PROCESS_TYPE` SHALL NOT be set in the process execution environment.
+* `/cnb/process` SHALL be removed from the beginning of `PATH`.
```

Comment:
Not blocking: I wonder if we should change this behavior. It would allow processes to execute other processes, which could be a huge win. But I'm not sure what happens when the launcher executes the launcher again?

Reply:
We agreed in the `RFC` to strip it so that process names like `java` wouldn't conflict with executables. I think we should stick with that plan for now b/c it seems like the safer choice.